src/matrix/room/ObservedEventMap.js

```diff
@@ -25,7 +25,7 @@ export class ObservedEventMap {
     observe(eventId, eventEntry = null) {
         let observable = this._map.get(eventId);
         if (!observable) {
-            observable = new ObservedEvent(this, eventEntry);
```

Comment:
what were these changes about? are they still desirable now that we didn't end up going with observeEvent?

Reply:
I believe that there's a bug here. If you try to unsubscribe from an observed event whose content is null (which is possible, for instance, if we can't retrieve an event from storage, or just haven't yet), you end up de-referencing a null when trying to remove the observedevent from the map (null.id).

Reply:
gotcha, good catch!